Dockerfile

```diff
@@ -28,6 +29,30 @@ RUN ldconfig
 
 WORKDIR /home/V2X-Hub/
 RUN /home/V2X-Hub/container/setup.sh
+
+# run final image
+FROM ubuntu:$UBUNTU_VERSION AS v2xhub
+ENV DEBIAN_FRONTEND=noninteractive
+ADD scripts/install_dependencies.sh /usr/local/bin/
+RUN /usr/local/bin/install_dependencies.sh
+
+COPY ./container /home/V2X-Hub/container/
+WORKDIR /home/V2X-Hub/container/
+RUN ./database.sh
+RUN ./library.sh
+RUN ldconfig
+
+COPY --from=dependencies /lib/systemd/system/tmxcore.service /lib/systemd/system/
+COPY --from=dependencies /usr/local/plugins/ /usr/local/plugins/
+COPY --from=dependencies /usr/local/lib/ /usr/local/lib/
+COPY --from=dependencies /usr/local/bin/ /usr/local/bin/
+COPY --from=dependencies /usr/sbin/tmxcore.service /usr/sbin/
+COPY --from=dependencies /var/www/plugins/ /var/www/plugins/
+COPY --from=dependencies /var/log/tmx/ /var/log/tmx/
```

Comment:
You are missing some dependencies here I think. I know some tmx utils stuff is installed in `/usr/local/share/tmx/`. Additionally, all new libraries that Rob and Cody worked on get installed in `opt/carma/cmake`. Lastly I think some modules from the `ext/` directory like the pedestrian REST api  are installed in `/usr/local/share/qserverPedestrian/cmake` and `/usr/local/share/v2xhubWebAPI/cmake/`. You can try running the pedestrian plugin and see if you encounter issues to confirm this.

Reply:
It might be easiest to just copy /opt/carma and /usr/local/share/

Reply:
Didn't run into any issues with the Ped plugin. Worked with both the websocket and rest configurations. Looking through this again to make sure I didn't miss any dependencies.

Reply:
@paulbourelly999 Addressed all of these. Let me know what you think.